storage/migrations/02_runtimes.up.sql

```diff
@@ -111,6 +111,7 @@ CREATE TABLE chain.runtime_events
 
   tx_hash HEX64,
   tx_eth_hash HEX64,
+  timestamp TIMESTAMP WITH TIME ZONE NOT NULL,
```

Comment:
I think that recently we started doing migrations the "proper" way. e.g. not updating existing migrations and instead creating a new migration that migrates the DB schema.  Since this added field is `NOT NULL` I assume there will also be the need to populate it when migrating live DBs. An added benefit of having a separate migration is that that SQL query for populating the timestamps will be committed and reviewed as well.
 
--- 

But I guess that for such simple cases, it's fine to also just update the old migration and manually do the migration when deploying. Not sure what the stance of the team is on this.

Reply:
Good point; I'll add the appropriate migration. Agreed that this can lead to a lot of tiny migrations that are hard to track - a good middle ground might be to always create "proper" migrations during development, but consolidate them into existing migrations prior to any full reindex deployment.

Reply:
Yeah short-term we used to be hackier, and now we're trying to be more proper and write actual migrations ... so thank you. Even if not strictly needed, it's "nice", and it also helps us build discipline.

For our own internal use, I'd still be fine with doing the migration manually (because it's low stakes and easy to revert), but now that we've launched more publicly we should ideally operate under the assumption that other people are running their own deployments of the indexer, and we cannot control those deployments and don't want to force a reindex.
That said, the number of external deployments is still very low and possibly zero at this point, so we can be a little pragmatic/flexible and rethink this on a case by case basis. 

**One request though:** Can you add a comment here about the column that you're adding via a separate migration? Like [we did here](https://github.com/oasisprotocol/nexus/blob/774ab49875f16315c9cd6afac855c21e0009fd56/storage/migrations/02_runtimes.up.sql#L224-L224) and in a few other places. It will make future consolidations, if we ever get to that, easier. It also makes it easier to glance the schema of any single table.

Reply:
Sure, added!